### PR TITLE
Switch to pyansys ci bot

### PR DESCRIPTION
**Pull request by PProfizi**
Example of working workflow:
https://github.com/ansys/pydpf-core/actions/runs/7568342361/job/20609355155

### CODE REVIEW

File: .github/workflows/ansys_lab.yml [hunk 1]
```diff
@@ -69,8 +69,8 @@ jobs:
       - name: "Push changes"
         shell: bash
         run: |
-          git config --global user.name "rlagha"
+          git config --global user.name "pyansys-ci-bot"
```
Comment (BClappe) [inline .github/workflows/ansys_lab.yml:72]: Did you configure the proper rights to the ci bot?

**Review by BClappe — APPROVED** [review]

File: .github/workflows/ansys_lab.yml (hunk 1 above)
Comment (PProfizi) [reply .github/workflows/ansys_lab.yml:72]: @BClappe it is managed at the organization level exactly for that type of task. I am not the one who gave permissions to the bot but I was told to use it, and considering an update pipeline passed, I'd say it works.

Comment (RobPasMue) [reply .github/workflows/ansys_lab.yml:72]: As long as the bot is authorized with "write" permissions in your repository, this token should work

Comment (RobPasMue) [reply .github/workflows/ansys_lab.yml:72]: Anytime!

**Pull request merged**